app/components/conferences/create/EventBasicDetails.js

```diff
@@ -172,6 +172,15 @@ export const EventBasicCreate = ({ setDraft, handleToast }) => {
               onChange={handleChange}
             />
           </Form.Group>
+          <Form.Group className="mb-3">
+            <Form.Label>Event Organizer logo</Form.Label>
+            <Form.Control
+              name="logo-url"
+              type="url"
+              placeholder=""
```

Comment:
Please add a placeholder with some link value. It will help the user know the correct format of data to be added to the form.

Thank you!

Reply:
Done! Please review the changes.
Thank You!